explorer/explorer.go

```diff
@@ -1,4 +1,7 @@
-package main
+// Package explorer handles the explorer subsystem for displaying the explorer pages
+// Copyright (c) 2017, The Dcrdata developers
```

Comment:
:+1: Let's update the LICENSE file too.
Although, I'm leaning toward all lowercase as our universal formatting, despite being a proper noun:
`The dcrdata.org developers` or `The dcrdata developers`